frontend/public/components/api-explorer.tsx

```diff
@@ -110,8 +110,10 @@ const Group: React.FC<{ value: string }> = ({ value }) => {
     <>{value}</>
   ) : (
     <>
```

Comment:
You can eliminate the fragment (`<>`) now that you've added the `<span className="co-break-word">` as the parent node.

Reply:
Resubmitted,thanks.

Reply:
Thanks, @orangeji11.  I tested locally and everything looks good.  

One final request, can you please squash your commits to a single commit?

Reply:
Sure,I'm done:)